Index: code-server/lib/vscode/src/vs/base/node/macAddress.ts
===================================================================
--- code-server.orig/lib/vscode/src/vs/base/node/macAddress.ts
+++ code-server/lib/vscode/src/vs/base/node/macAddress.ts
@@ -1,25 +1,3 @@
-/*---------------------------------------------------------------------------------------------
- *  Copyright (m) Microsoft Corporation. All rights reserved.
- *  Licensed under the MIT License. See License.txt in the project root for license information.
- *--------------------------------------------------------------------------------------------*/
-
-import { networkInterfaces } from 'os';
-
-export function getMac(): string {
-	const ifaces = networkInterfaces();
-	for (const name in ifaces) {
-		const networkInterface = ifaces[name];
-		if (networkInterface) {
-			for (const { mac, internal } of networkInterface) {
-				if (!internal && mac !== '00:00:00:00:00:00') {
-					return mac;
-				}
-			}
-		}
-	}
-	throw new Error('Unable to retrieve mac address (unexpected format)');
-}
+export function getMac(): string {
+	return '00:00:00:00:00:00';
+}
